lib/hammer_cli_foreman_discovery.rb

```diff
@@ -1,3 +1,5 @@
 module HammerCLIForemanDiscovery
   require 'hammer_cli_foreman_discovery/discovery'
+  require 'hammer_cli_foreman_discovery/discovery_rule'
```

Comment:
Could you load the subcommands lazily? It helps to speed up hammer. Similar to what we do in https://github.com/theforeman/hammer-cli-foreman/blob/master/lib/hammer_cli_foreman.rb#L30